README.md

```diff
@@ -55,7 +57,7 @@ Remove all the completed todos after the `Clear completed` button click:
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Grifion.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
```

Comment:
Add this demo link to the PR description 
<img width="931" alt="image" src="https://github.com/user-attachments/assets/ac8f845e-32ca-4be5-bb96-ea02a489d8ba" />